node/forest_libp2p/src/blocksync/message.rs

```diff
@@ -22,11 +24,82 @@ pub struct BlockSyncRequest {
     pub options: u64,
 }
 
+impl BlockSyncRequest {
+    /// If a request expects blocks to be included in response.
+    pub fn include_blocks(&self) -> bool {
+        self.options == BLOCKS || self.options == BLOCKS_MESSAGES
+    }
+
+    /// If a request expects messages to be included in response.
+    pub fn include_messages(&self) -> bool {
+        self.options == MESSAGES || self.options == BLOCKS_MESSAGES
+    }
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum BlockSyncResponseStatus {
```

Comment:
doc comment would be nice, nbd tho

Reply:
ya, thought about it, but couldn't squeeze anything less obvious than "Status of the BlockSync response" haha. Let me try once again.

Reply:
updated